test/multi_node_test_2.py

```diff
@@ -34,6 +43,30 @@ def test_cluster():
     assert not any(node.any_processes_alive() for node in g.list_all_nodes())
 
 
+def test_internal_config():
```

Comment:
Don't we want to use the internal config to speed up the other tests in this file?

Reply:
Ah never mind, I see you've already done that.